frame/contracts/src/lib.rs

```diff
@@ -177,8 +177,12 @@ pub mod pallet {
 	use frame_system::pallet_prelude::*;
 
 	/// The current storage version.
+	#[cfg(not(test))]
 	const STORAGE_VERSION: StorageVersion = StorageVersion::new(9);
 
+	#[cfg(test)]
+	const STORAGE_VERSION: StorageVersion = StorageVersion::new(2);
```

Comment:
this is so we can write test on the migration that depends on the current storage_version

Reply:
Yep, maybe I can find a way to create a mock pallet instead in migration so we don't need this altogether